src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php

```diff
@@ -75,11 +75,7 @@ public function __construct($foreground = null, $background = null, array $optio
     }
 
     /**
-     * Sets style foreground color.
-     *
-     * @param string|null $color The color name
-     *
-     * @throws InvalidArgumentException When the color name isn't defined
```

Comment:
should we move the `@throw`s to the contract instead?

Reply:
yes updated